plugins/filetrees/yazi.nix

```diff
@@ -0,0 +1,161 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
```

Comment:
Do we need `with lib` here? I know we've done this historically, but I'd rather inherit the few things we use repeatedly and access the rest explicitly.